packages/notifier/src/storesub.js

```diff
@@ -59,13 +59,13 @@ export const makeStoredSubscriber = (subscriber, storageNode, marshaller) => {
 
   /** @type {Unserializer} */
   const unserializer = Far('unserializer', {
-    unserialize: E(marshaller).unserialize,
+    unserialize: data => E(marshaller).unserialize(data),
```

Comment:
would it be reasonable to have a utility to attenuate a far object? This is basically passing on a subset of `marshaller`.

Reply:
Sure, but I don't know if the complexity of such a generic tool would be amortized?

Reply:
This is a good suggestion. The loss of autobinding raises the notational cost of manually attenuating, thereby potentially raising the comparative benefit of such a utility. Let's keep this in mind.